operations/app/src/environments/tf

```diff
@@ -165,6 +183,18 @@ if [[ "${ACTUAL_RESULTS?}" == "${EXPECTED_RESULTS?}" ]]; then
         ACTUAL_RESULTS="${ACTUAL_RESULTS?}${?}"
         ;;
 
+    plan-file*)
```

Comment:
why `plan-file` instead of `plan`? I would stick to the tf commands as close as possible; it is just a wrapper after all

Reply:
`plan-file` runs a number of commands in addition to `plan`